signalfx/client.py

```diff
@@ -23,13 +24,19 @@ class BaseSignalFx(object):
 
     def __init__(self, api_token, ingest_endpoint=DEFAULT_INGEST_ENDPOINT,
                  api_endpoint=DEFAULT_API_ENDPOINT, timeout=DEFAULT_TIMEOUT,
-                 batch_size=DEFAULT_BATCH_SIZE, user_agents=[]):
+                 batch_size=DEFAULT_BATCH_SIZE, user_agents=None,
+                 enable_aws_unique_id=False):
```

Comment:
Ofcourse a user should be able to override it by passing his own flag. I was asking more like def __init__(..., enable_aws_unique_id=DEFAULT_ENABLE_UNQ_ID) so the flag is not inside the code but somewhere more accessible.